benchmark/fs/bench-accessSync.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+const tmpdir = require('../../test/common/tmpdir');
+tmpdir.refresh();
+
+const tmpfile = tmpdir.resolve(`.existing-file-${process.pid}`);
+fs.writeFileSync(tmpfile, 'this-is-for-a-benchmark', 'utf8');
+
+const bench = common.createBenchmark(main, {
+  type: ['existing', 'non-existing', 'non-flat-existing'],
```

Comment:
I'd include the constants `fs.constants.F_OK` and `fs.constants.W_OK` in the list. It shouldn't be that different for this benchmark tough. Feel free to ignore if the numbers show a similarity.

Reply:
I agree. I prefer a follow up pull request if this is not a blocker from your side.